sale_promotion_discount_in_field/tests/test_promotion_discount_in_field.py

```diff
@@ -2,11 +2,9 @@
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 
 from odoo.exceptions import UserError
-from odoo.tests import tagged
 from odoo.tests.common import SavepointCase
 
 
-@tagged("post_install", "-at_install")
 class CetmixTestSaleCouponProgram(SavepointCase):
```

Comment:
You can't put a company name in the test class

Reply:
didn't want to add to the diff, but you're right. Fixed.